result/bgpprefix.yaml

```diff
@@ -281,3 +303,340 @@ components:
+          description: >-
+            The Local Administrator sub-field contains a number from a numbering space that is administered by the organization to which the
+            Autonomous System number carried in the Global Administrator sub-field has been assigned by an appropriate authority.
+          x-field-uid: 2
+    BgpPrefix.ExtendedCommunity.Transitive4OctetAsType.RouteOrigin:
+      description: >-          
+        The Route Origin Community identifies one or more routers that inject a set of routes (that carry this Community) into BGP.
+        It is sent with sub-type as 0x03.
+      type: object
+      properties:
+        global_4byte_as:
+          description: >-
+            The four octet IANA assigned AS value assigned to the Autonomous System.
+          type: integer
+          format: uint32             
+          x-field-uid: 1
+        local_2byte_admin:
+          description: >-
+            The Local Administrator sub-field contains a number from a numbering space that is administered by the organization to which the
+            Autonomous System number carried in the Global Administrator sub-field has been assigned by an appropriate authority.
+          type: integer
+          format: uint32       
+          maximum: 65535
+          x-field-uid: 2
+    BgpPrefix.ExtendedCommunity.Transitive4OctetAsType:
+      description: >-
+        The Transitive Four-Octet AS-Specific Extended Community is sent as type 0x02. It is defined in RFC 5668.       
+      type: object
+      properties:        
+        choice:
+          type: string     
+          x-enum:
+            route_target_subtype:
+              x-field-uid: 1
+            route_origin_subtype:
+              x-field-uid: 2
+          x-field-uid: 1
+        route_target_subtype:
+          x-field-uid: 2
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.Transitive4OctetAsType.RouteTarget'
+        route_origin_subtype:
+          x-field-uid: 3
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.Transitive4OctetAsType.RouteOrigin'
+    BgpPrefix.ExtendedCommunity.TransitiveOpaqueType.Color:
+      description: >-
+        The Color Community contains locally administrator defined 'color' value which is used in conjunction with Encapsulation 
+        attribute to decide whether a data packet can be transmitted on a certain tunnel or not.
+        It is defined in RFC9012 and sent with sub-type as 0x0b.             
+      type: object
+      properties:
+        flags:
+          description: >-
+            Two octet flag values. 
+          type: integer
+          format: uint32         
+          maximum: 65535              
+          x-field-uid: 1
+        color:
+          type: integer
+          format: uint32         
+          description: >-
+            The color value is user defined and configured locally and used to determine whether a data packet can be transmitted on a certain tunnel or not
```

Comment:
pls check the additional newline

Reply:
These are github specific views and probably not in our control. Redocli appears to show them correctly.